shared/src/validation-schemas/user-details/user-details-update-info.validation-schema.ts

```diff
@@ -26,6 +26,11 @@ const userDetailsUpdateInfo = Joi.object({
     .messages({
       'string.empty': UserDetailsValidationMessage.GENDER_REQUIRE,
     }),
+  [getNameOf<UserDetailsUpdateInfoRequestDto>('dateOfBirth')]: Joi.date()
```

Comment:
can be `date`(string) on `null`. an empty string isn't allowed. should be optional